integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -462,4 +467,106 @@ describe("/admin/product-categories", () => {
       )
     })
   })
+
+  describe("POST /admin/product-categories/:id/products/batch", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      productCategory = await simpleProductCategoryFactory(dbConnection, {
+        id: "test-category",
+        name: "test category",
+      })
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("should add products to a product category", async () => {
+      const api = useApi()
+      const testProduct1 = await simpleProductFactory(dbConnection, {
+        id: "test-product-1",
+        title: "test product 1",
+      })
+
+      const testProduct2 = await simpleProductFactory(dbConnection, {
+        id: "test-product-2",
+        title: "test product 2",
+      })
+
+      const payload = {
+        product_ids: [{ id: testProduct1.id }, { id: testProduct2.id }],
```

Comment:
thought: I don't think we have a real convention about that. But at some point we should find something. It seams weird that have product_ids with a collection of object that potentially in the future can contains something else that the id. I would say it could be named products_data or something like that but let's discuss that point in the future with the team 😄

Reply:
I would've chosen just an array of IDs, but alas, convention.